packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -75,6 +75,18 @@ $isPaddedMargin: 24px;
   &.cellIsEmpty {
     text-overflow: clip;
   }
+
+  &:hover .GripperBarVerticalStyle {
+    display: block;
+   }
+}
+
+.GripperBarVerticalStyle {
```

Comment:
nit: can we change to start with lowcase to match others?

Reply:
Fixed